contracts/src/v0.8/dev/interfaces/OptimismGasInterface.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+/**
+ * @notice OptimismGasInterface provides an interface to estimate optimism L1 fee in wei
+ */
+interface OptimismGasInterface {
```

Comment:
nit. can we call it OVM_GasPriceOracle to keep it consistent with the actual contract

Reply:
Done